app/controllers/api/doughnuts_controller.rb

```diff
@@ -1,6 +1,8 @@
 class Api::DoughnutsController < ApplicationController
   def index
     @doughnuts = Doughnut.all
+
+    render json: @doughnuts
   end
 
   def show
```

Comment:
Also need to edit this method to give a JSON-formatted response of the Doughnut entity

Reply:
We can also delete the the following methods here, since I don't believe we will be supporting these functionalities:
1. `new`
2. `edit`
3. `update`
4. `delete`

Reply:
We will have delete though right.